apps/web/src/containers/WalletContainer.ts

```diff
@@ -23,16 +24,51 @@ import { createContainer } from 'unstated-next'
 
 import { Web3Container } from './Web3Container'
 
-export type WalletBalance = ReservesData & {
-  balance: string
-  icon: string
+type DefaultReserveData = {
+  id: string
+  name: TokenList
   fullName: string
+  symbol: TokenList
+  icon: string
+  decimals: number
+  balance: string
   supplyAPY: number
+  liquidityRate: number
+}
+export type ReservesData = DefaultReserveData & {
   variableBorrowAPY: number
+  totalATokenSupply: number
+  totalCurrentVariableDebt: number
+  variableBorrowRate: number
+  underlyingAsset: string
+}
+
+export type UserReserveData = DefaultReserveData & {
+  currentATokenBalance: number
+  currentVariableDebt: string
+  currentTotalDebt: string
+  usageAsCollateralEnabledOnUser: boolean
+}
+
+export const initailReserveData = {
```

Comment:
you can move this to the Modal if you are using it only there, if not, you can type this.

```tsx
export const initailReserveData: ReserveData = {
```

Reply:
done